app/scripts/modules/azure/serverGroup/configure/serverGroupConfiguration.service.js

```diff
@@ -143,6 +143,22 @@ module.exports = angular
         return result;
       }
 
+      function configureZones(command) {
+        const result = { dirty: {} };
+        const filteredData = command.backingData.filtered;
+        if (command.region === null) {
```

Comment:
Should this also return early if `command.region` is `undefined`?  If so I suggest changing this to:

```
if (!command.region) {
  return result;
}
```